channels/test/liveserver.py

```diff
@@ -0,0 +1,205 @@
+import multiprocessing
+import os
+import sys
+
+import django
+from daphne.server import Server
+from django.core.exceptions import ImproperlyConfigured
+from django.test.testcases import TransactionTestCase
+from django.test.utils import modify_settings, override_settings
+from django.utils import six
+
+from .. import DEFAULT_CHANNEL_LAYER
+from ..asgi import ChannelLayerManager
+from ..worker import Worker, WorkerGroup
+
+# NOTE: We use ChannelLayerManager to prevent layer instance sharing
+# between forked process.  Some layers implementations create
+# connections inside the __init__ method.  After forking child
+# processes can lose the ability to use this connection and typically
+# stuck on some network operation.  To prevent this we use new
+# ChannelLayerManager each time we want to initiate default layer.
+# This gives us guaranty that new layer instance will be created and
+# new connection will be established.
```

Comment:
I know this may be seen as a stupid comment, but can you move lines 16-23 into a module docstring on top of the file? I think this gets read more often that way.

If it's really about the `from ..asgi import ChannelLayerManager` a single line comment on line 12 above should suffice (in addition, when the comment block is available as a docstring).